include/configcontainer.h

```diff
@@ -82,9 +84,11 @@ class ConfigContainer : public ConfigActionHandler {
 
 	static const std::string PARTIAL_FILE_SUFFIX;
 
+	/// Returns the reference to the Rust object.
+	const configcontainer::bridged::ConfigContainer& get_rust_ref() const;
```

Comment:
It looks like this is unused and can be removed.

If it *is* used, let's discuss how to get rid of it :)